x/gamm/client/cli/cli_test.go

```diff
@@ -107,6 +108,17 @@ func (s *IntegrationTestSuite) TestNewCreatePoolCmd() {
 	)
 	s.Require().NoError(err)
 
+	okJSON := testutil.WriteToNewTempFile(s.T(), fmt.Sprintf(`
+	{
+	  "%s": "1node0token,3stake",
+	  "%s": "100node0token,100stake",
+	  "%s": "0.001",
+	  "%s": "0.001"
+	}
+	`, cli.FlagWeights, cli.FlagInitialDeposit, cli.FlagSwapFee, cli.FlagExitFee))
+
+	badJSON := testutil.WriteToNewTempFile(s.T(), "bad json")
```

Comment:
Can we have some badJSON's with invalid types. (e.g. just putting 0.001, no quotations)